docs/compatibility_matrix.md

```diff
@@ -0,0 +1,14 @@
+# Compatibility Matrix
```

Comment:
```suggestion
# Scalar DB Supported Databases
```

Reply:
Thank you for your suggestion.
Suggestion integrated.